dietpi/dietpi-software

```diff
@@ -4569,14 +4569,14 @@ _EOF_
 
 			Banner_Installing
 
-			#Binary URL fails connection test, so we need to parent back a little: https://github.com/Fourdee/DietPi/issues/445#issuecomment-283400449
-			INSTALL_URL_ADDRESS='https://github.com/blynkkk/blynk-server/releases'
+			INSTALL_URL_ADDRESS='https://api.github.com/repos/blynkkk/blynk-server/releases/latest'
 			G_CHECK_URL "$INSTALL_URL_ADDRESS"
 
-			mkdir -p /etc/blynkserver
-
-			INSTALL_URL_ADDRESS='https://github.com/blynkkk/blynk-server/releases/download/v0.36.2/server-0.36.2-java8.jar'
-			wget "$INSTALL_URL_ADDRESS" -O /etc/blynkserver/server.jar
+            no_check_url=$(curl -s $INSTALL_URL_ADDRESS | grep -m1 'browser_download_url' | cut -d \" -f 4)	
```

Comment:
@joaofl 
`no_check_url` is meant as an option for the `Download_Install` function. Currently not sure if this will break something, if set to an URL, but please use `INSTALL_URL_ADDRESS` here. It can be overwritten.

Also check alignment, use three tabs instead if spaces. Some for some lines below.